src/AzureClient/EntryPoint/EntryPoint.cs

```diff
@@ -252,11 +292,27 @@ public Task<IQuantumMachineJob> SubmitAsync(IQirSubmitter submitter, AzureSubmis
 
             var entryPointInput = GetEntryPointInputArguments(submissionContext, allowOptionalParameters);
             Logger?.LogInformation("Submitting job {FriendlyName} with {NShots} shots.", submissionContext.FriendlyName, submissionContext.Shots);
-            var options = SubmissionOptions.Default.With(submissionContext.FriendlyName, submissionContext.Shots, submissionContext.InputParams);
+            var options = SubmissionOptions.Default.With(
+                friendlyName: submissionContext.FriendlyName, 
+                shots: submissionContext.Shots,
+                inputParams: submissionContext.InputParams,
```

Comment:
Just for my understanding, inputParams here does not equal inputParams on the job object right? Is this technically for arguments?

Or is this a grab-bag of other params that will get merged with friendlyName, shots, and targetCapability in a later submit command?

Reply:
It's a `ImmutableDictionary<string, string>` grab-bag.

From the docs:
> Provider-specific job parameters to be passed to the execution target, expressed as one or more JSON {"key":"value",...} pairs.
> These parameters apply to <c>%azure.execute</c> and <c>%azure.submit</c>. The JSON may not contain separating spaces when used in a Jupyter notebook. 
> NOTE: This property was named "InputParams" (instead of "JobParams") because the closely
   related implementation in microsoft/qsharp-runtime used the name "InputParams"
   (see https://github.com/microsoft/qsharp-runtime/pull/829).
> Please notice the difference between "InputParams" and the preexisting "InputParameters" member below.

Reply:
`options.InputParams` is related to `inputParams` on the job object but it is adding to the job object (rather than setting `inputParams`).